[FEATURE] Pouvoir voir le détail d'un test statique sur PixEditor (PIX-7796)

laura-bergoens opened the pull request:
## :unicorn: Problème
EPIC tests statiques sur PixAdmin, après la liste des tests on veut pouvoir cliquer sur un item de la liste et être redirigé sur la page de détails.

## :robot: Solution
- Ajout d'une route API GET /api/static-courses/:id
- Affichage selon le figma
- Réorganisation CSS et de templates

## :rainbow: Remarques


## :100: Pour tester
Se rendre dans le menu des tests statiques > cliquer sur un test statique et regarder le résultat.
Essayer les différents boutons de prévisualisation etc...

pix-bot-github commented:
Une fois l'application déployée, elle sera accessible à cette adresse https://pix-lcms-review-pr121.osc-fr1.scalingo.io
Les variables d'environnement seront accessibles sur scalingo https://dashboard.scalingo.com/apps/osc-fr1/pix-lcms-review-pr121/environment

laura-bergoens committed:
Add API route to get a specific static course by ID

laura-bergoens committed:
move routes that belong to static courses into a common folder

laura-bergoens committed:
setup details page

laura-bergoens committed:
reorganize DOM under static courses features

laura-bergoens committed:
fix button borders and bug when clicking row

jujumathieu commented:
- Liste des tests : 
        - Le bouton qui permet de copier le lien de Prévisualisation reste appuyé suite au clic.. 
        - Le bouton de Prévisualisation reste en hover une fois cliqué.
        - La barre de menu doit restée fixe quand on descend dans le tableau (lorsqu’il y a plus de 10 éléments affichés dans le tableau)
    - Détail d’un test
        - Fil d’ariane, finalement on passera par un nouveau ticket ? (Finalement il y a le bouton « retour »)
        - L’affichage du statut du test, à traiter dans un nouveau ticket ?
        - L’Icône du bouton de Prévisualisation est collée au texte
        - Suite au clic sur bouton « Prévisualisation », celui-ci reste enfoncé (ok pour le bouton de copie du lien)
        - Encart Liste des épreuves :
            - La largeur de la colonne « # » peut éventuellement être diminuée
            - Le # d’épreuve qui devrait commencer à 1
            - La consigne est finalement non tronquée mais affichée sur plusieurs lignes. Il faudrait peut être mettre une certaine limite dans le cas de consignes très longues.. (ça pourra être fait via un autre ticket)
            - Les acquis chevauchent quelque fois la colonne « statut »

laura-bergoens committed:
fixes after review

challenge index
space between icon and text in button links
word wrap and ellipsis in table

laura-bergoens committed:
fix column sizes in table

laura-bergoens committed:
fix menu button not following when scrolling
